src/naip_cnn/models.py

```diff
@@ -39,3 +39,24 @@ def CNN_v1(
     model.compile(loss="mean_squared_error", optimizer=optimizer, metrics=["mae"])
 
     return model
+
+
+def UNet(

```

Comment:
How much do you think this new model (UNet) played into the increased accuracy?  Did you try to run with `CNN_v1` as well with the 5x5 labels?  Pefhaps that model doesn't support the image labels?

Reply:
Great question! That's not something I'd tested until now because `CNN_v1` wasn't directly compatible with 2D outputs, and I also wanted to get away from the `Conv3D` layer that's typically used for video data, but after making a few modifications (changing the last `Dense` layer output shape to 25 and adding a `Reshape` layer) I was able to get it to train. 

**Using `CNN_v1` with 2016 cover data, I was able to get an MSE of 223 and an R^2 of 0.54**, which are essentially the same as we were getting with the 1D labels. It looks like the updated model might be the main difference we're seeing here, although I'll have to try training the new model with the old 1D data to confirm that - it could be a combination of the two changes. I'll run that and report back here!

Reply:
Awesome summary table at the top.  Very clear how it all plays out!  UNet + 2D prediction + lots of samples for the win ...